framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -65,7 +67,9 @@
   private Connection connection = null;
   private ConnectionPool connectionPool = null;
   private String commitId;
-  private String version;
+  private static String version;
+  private String[] injectionKeys = {"DRILL_VERSION"};
```

Comment:
Will change version to static final. enum type is not suited here. It will work well with the switch statement but not work well with String.replaceAll(). In addition, the injections HashMap needs String type as key.